packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -120,13 +120,36 @@ const changeConnectionState = (
 	runtime.setConnectionState(connected, clientId);
 };
 
+type ISignalEnvelopeWithClientIds = ISignalEnvelope & {
+	clientId: string;
+	targetClientId?: string;
+};
+
```

Comment:
We're adding stuff to envelopes? That isn't something that is happening outside of test, right? So why is it something we want to do in test?